### DISCUSSION TITLE
Calibration of PH Probe?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#973] [REM] [answered]
I just installed a new AS, consumer grade PH probe.   It reads 7.4 when a Taylor reagent test says 7.6.

I noticed a calibration menu in REM, but only to calibrate for temperature.  Is there someplace similar for ph probes?

If not, I'll do comparisons against a Taylor test for a few days, then just adjust my setpoint higher.

**Comment by rstrouse** [comment, marked as answer]
Here you go.

https://github.com/tagyoureit/nodejs-poolController/wiki/DIY-REM-Chem-Controller#calibrate-the-ph-probe

**Comment by davthomaspilot** [comment]
Wow, thanks!

**Comment by davthomaspilot** [comment]
I think I asked this previously, but aren't we only interested in the accuracy of the ph probe for values close to the set point?  Unless things aren't working, that's where the ph should always be.

And, if it's not, do we really care if the pH is 4.0 versus 5.0?  Or 8.0 versus 9.0?  All the controller does with the ph reading is dose or not dose, right?

Related question--

I was surprised to discover that a new Atlas Scientific ph probe read .2 lower than what I get with Taylor Reagent measurement (7.6 versus 7.8).   I'd think the probes would be shipped to a calibrated value much closer than that.

**Reply by tagyoureit** [reply]
The probes offer 1- and 3-point calibrations.  3 points will be a more fitting curve across the entire spectrum while 1-point will be most accurate for the single measurement.  

> I was surprised to discover that a new Atlas Scientific ph probe read .2 lower than what I get with Taylor Reagent measurement (7.6 versus 7.8). I'd think the probes would be shipped to a calibrated value much closer than that.

How do you know which one is incorrect?  :). Just because you've been using the Taylor kit longer, maybe that one is off.  But, regardless, to your larger question, the precision of the measurements isn't what's paramount.  If your pool looks great and the water feels great at 7.6, then go with that.  If the water looks good and feels great at 7.4, then go with that.  Are you even able to tell the difference in your pool at 7.4 vs 7.6?  

The pH (and ORP, for that matter), are really just yardsticks for keeping the chemicals as balanced as possible.

**Comment by davthomaspilot** [comment]
Brand new reagents for the Taylor kits.
 Are you even able to tell the difference in your pool at 7.4 vs 7.6?

Absolutely--an extra gallon of acid each week to keep it at 7.6 versus 7.8.  Even 7.6 takes a lot of acid.  Hence my interest in seeing how low I really have to keep the pH.
I would target even higher, except that REM won't let me set ph lock-out high enough.

Thanks!    On Friday, June 28, 2024 at 04:34:59 PM EDT, tagyoureit ***@***.***> wrote:  
 
 


The probes offer 1- and 3-point calibrations. 3 points will be a more fitting curve across the entire spectrum while 1-point will be most accurate for the single measurement.


I was surprised to discover that a new Atlas Scientific ph probe read .2 lower than what I get with Taylor Reagent measurement (7.6 versus 7.8). I'd think the probes would be shipped to a calibrated value much closer than that.


How do you know which one is incorrect? :). Just because you've been using the Taylor kit longer, maybe that one is off. But, regardless, to your larger question, the precision of the measurements isn't what's paramount. If your pool looks great and the water feels great at 7.6, then go with that. If the water looks good and feels great at 7.4, then go with that. Are you even able to tell the difference in your pool at 7.4 vs 7.6?

The pH (and ORP, for that matter), are really just yardsticks for keeping the chemicals as balanced as possible.

**Reply by rstrouse** [reply]
You can set the pH lockout to a range of 7.2 to 8.4.  The reason this exists is that an extremely high pH will disrupt any readings you have for sanitizer and make and confuse an ORP probe.  Heck it will also confuse your test kit as well because of the buffering of Hypochlorous acid.  How high do you need the lockout to be?

Calibration of that probe should bring it in line over our measurement to an accuracy way beyond the calibration of your eye.  That is if your calibration solutions are up to par.  The factory can never accurately, calibrate your probe because of the small differences in wiring (capacitance) that will naturally occur in your connections... etc.

I have found that during the cooler months a pH of 7.8 gives me the right CSI that I am looking for.   When it warms up I push that down to 7.6.  All of this is dependent on your Calcium Hardness and do not chase the Total Alkalinity.  It will drive you insane.  The primary reason the TA is required in pools are because addition of acid has a quick reaction in pools and simple things like pucks containing acid or performing manual dosing can drop the bottom out of pH.  The TA is there to ensure it doesn't push too low for too long.

**Reply by davthomaspilot** [reply]
I understand the purpose of pH lockout and think it's a must have feature...

How high do you need the lockout to be?

I would like to be able to disable it entirely.  

Until last week, I had a bad ph probe.  So, even though I manually monitored and dosed acid, I could not automatically dose chlorine due to the ph lockout feature.

Also, we spend a month each summer out of town.  We've run out of acid.    A couple of days after ph lockout occurs it's guaranteed we'll have a green pool.  While the chlorine isn't as effective when the ph is off, it's better wasting some chrlorine rather than just letting it go.  Better to have high ph and "shocking" until the acid is delivered rather than having 0 acid AND 0 chlorine being dosed.

I now have a 15 gallon acid tank along side the 15 gallon chlorine tank, so we shouldn't run out while out this year while we are out of town.  But, it would be a nice feature, IMHO, to be able to disable it for "special cases".

Thanks!

**Reply by tagyoureit** [reply]
I don't think we should be changing the lockout feature for the reasons @rstrouse says.  If you need to "get around" the ph lockout because the chemicals are so out of whack, you can disconnect the pH readings from njsPC so it doesn't keep reading above 8.4.  (You may need to stop njsPC, manually edit the chemController->ph->probe->level reading in poolState.json, and then restart.)
